### PR TITLE
OJ-1883-added mocked jest tests

### PR DESCRIPTION
**Pull request by rahulGDS**
### Issue tracking

- [OJ-1883](https://govukverify.atlassian.net/browse/OJ-XXXX)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1883]: https://govukverify.atlassian.net/browse/OJ-1883?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: integration-tests/events/sfn_valid_input.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "nino": "AA000003D",
```
Comment (surajkumar) [inline integration-tests/events/sfn_valid_input.json:2]: `nino` not required as an input for this state machine

File: integration-tests/tests/mocked/get-question/step-functions-constants.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import fs from "fs";
+import path from "path";
+
+const EVENT_FILE = path.join(__dirname, "../../../events/sfn_valid_input.json");
```
Comment (surajkumar) [inline integration-tests/tests/mocked/get-question/step-functions-constants.ts:4]: Check this path, might need to go up 1 more level

File: integration-tests/tests/mocked/get-question/step-functions-constants.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const STATE_MACHINE_FILE = path.join(
+  __dirname,
+  "../../../../step-functions/get-question.asl.json"
```
Comment (surajkumar) [inline integration-tests/tests/mocked/get-question/step-functions-constants.ts:7]: Check this path, might need to go up 1 more level

File: integration-tests/tests/mocked/get-question/step-functions-local.test.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+describe("step-function-local", () => {
+  describe("HMRC KBV Get Questions", () => {
+    let stnContainerHelper: SfnContainerHelper;
+    describe("is successful", () => {
+      beforeAll(async () => {
+        stnContainerHelper = new SfnContainerHelper();
+      });
+      afterAll(async () => stnContainerHelper.shutDown());
+      it("has a container is running", async () => {
+        expect(stnContainerHelper.getContainer()).toBeDefined();
```
Comment (surajkumar) [inline integration-tests/tests/mocked/get-question/step-functions-local.test.ts:15]: We should probably create a Jira ticket to address this

File: integration-tests/tests/mocked/get-question/step-functions-local.test.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+      });
+
+      describe("happy Case Scenario", () => {
+        it("should get 200 when session id present", async () => {
```
Comment (surajkumar) [inline integration-tests/tests/mocked/get-question/step-functions-local.test.ts:19]: This is testing more than just the `sessionId` being present. Currently, this is your only happy path test testing the entire flow, so can we rename this to be more accurate to that?

Comment (surajkumar) [reply integration-tests/tests/mocked/get-question/step-functions-local.test.ts:19]: Missing the HTTP 204 happy path test

File: integration-tests/tests/mocked/get-question/step-functions-constants.ts (hunk 3 above)
Comment (rahulGDS) [reply integration-tests/tests/mocked/get-question/step-functions-constants.ts:7]: No just checked it is correct

File: integration-tests/tests/mocked/get-question/step-functions-local.test.ts (hunk 4 above)
Comment (rahulGDS) [reply integration-tests/tests/mocked/get-question/step-functions-local.test.ts:15]: https://govukverify.atlassian.net/browse/OJ-1897

**Review by surajkumar — APPROVED** [review]

**Pull request merged**